### ISSUE TITLE
Form validation classes do not work when using custom wrappers

### ISSUE THREAD

**Issue by tswonke** [Laravel-Backpack/CRUD#1521]
When using custom wrapper attributes, the validation markers (and new asterisk) do not work. I tried to fix it as it only seems to be one line in `field_wrapper_attributes.blade.php`, but I ran into an issue:

```
@if (isset($field['wrapperAttributes']))
    @foreach ($field['wrapperAttributes'] as $attribute => $value)
        @if (is_string($attribute))
            {{ $attribute }}="{{ $value }}"
        @endif
    @endforeach

    @if (!isset($field['wrapperAttributes']['class']))
        class="form-group col-xs-12{{ isset($field['attributes']['required'])||$crud->isRequired($field['name'], $action) ? ' required' : '' }}"
    @else
        // works
        test="break it"
        // not working
        class="{{ $field['wrapperAttributes']['class'] }} {{ isset($field['attributes']['required'])||$crud->isRequired($field['name'], $action) ? ' required' : '' }}"
    @endif
@else
    class="form-group col-xs-12{{ isset($field['attributes']['required'])||$crud->isRequired($field['name'], $action) ? ' required' : '' }}"
@endif
```

I assumed that we only need that additional `@else` but this did not work for me, because `class` was already defined and it seems to be impossible to overwrite.

Any suggestions?

My would be outsource this tricky logic to php and only have an accessor for the wrapper attributes...

**Comment by tswonke** [comment]
#1523

**Comment by tabacitu** [comment]
You're totally right @tswonke - that DID happen, I had no idea. Thanks a lot for raising the issue AND providing the solution - you rock.

I just merged your PR, it'll be available with a ```composer update``` as 3.4.22 in a few hours, after I bundle it with a few more changes.

Cheers!
